typescript-timers/interval.ts

```diff
@@ -0,0 +1,22 @@
+const $timer = document.querySelector('.countdown-display');
+if (!$timer)
+  throw new Error('The class countdown-display did not load correctly');
+
+let count = 4;
+const intervalId = setInterval(function () {
+  count--;
+  switch (count) {
+    case 3:
+      $timer.textContent = '3';
+      break;
+    case 2:
+      $timer.textContent = '2';
+      break;
+    case 1:
+      $timer.textContent = '1';
+      break;
+    case 0:
+      $timer.textContent = '~Earth Beeeelooowww Us~';
```

Comment:
Do you need a case statement here? What do you really need to know for this check?